src/core/QCheck.ml

```diff
@@ -946,9 +966,10 @@ module Observable = struct
     type 'a t = 'a -> 'a -> bool
 
     let int : int t = (=)
+    let bytes : bytes t = (=)
     let string : string t = (=)
     let bool : bool t = (=)
-    let float : float t = (=)
+    let float : float t = Float.equal
```

Comment:
I think it would be cleaner to just submit the two small `float` equality fixes as a separate PR.
It will make it easier to track that fix when decoupled from this larger PR.

Reply:
That makes sense :+1: